uniffi_bindgen/src/bindings/python/gen_python/mod.rs

```diff
@@ -298,6 +298,31 @@ impl<'a> TypeRenderer<'a> {
             });
         ""
     }
+
+    // An inefficient algo to return type aliases needed for custom types
+    // in an order such that dependencies are in the correct order.
+    // Eg, if there's a custom type `Guid` -> `str` and another `GuidWrapper` -> `Guid`,
+    // it's important the type alias for `Guid` appears first.
+    fn get_custom_type_aliases(&self) -> Vec<(String, &Type)> {
+        let mut ordered = vec![];
+        for type_ in self.ci.iter_types() {
+            if let Type::Custom { name, builtin, .. } = type_ {
+                match ordered
+                    .iter()
+                    .position(|x: &(&str, &Type)| *name == x.1.as_codetype().type_label())
+                {
+                    // This 'name' appears as a builtin, so we must insert our type first.
+                    Some(pos) => ordered.insert(pos, (name, builtin)),
+                    // Otherwise at the end.
+                    None => ordered.push((name, builtin)),
+                }
+            }
+        }
```

Comment:
I think this could maybe fail if there were multiple levels of builtins, eg:

```
A { builtin: C},
B { }
C { builtin: B }
```

would get sorted to 

```
C { builtin: B }
A { builtin: C},
B { }
```

That seems pretty pathological for custom types and I think it's probably fine to ignore.  How about adding a note mentioning this?  That way we don't repurpose the algo for a situation where it would matter.

Reply:
Yeah, thanks!